usr/local/www/includes/functions.inc.php

```diff
@@ -346,4 +347,136 @@ function get_interfacestatus() {
 	return $data;
 }
 
+function get_nutstats() {
+	global $config;
+	$data = "";
+	$cmd = "";
+	$nut_config = $config['installedpackages']['nut']['config'][0];
```

Comment:
You should check isset($config['installedpackages']['nut']['config'][0]) to avoid possible PHP warnings when package was never installed on the system.